eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -164,6 +157,7 @@ jobs:
       - template: ../../../common/pipelines/templates/steps/verify-links.yml
         parameters:
           Directory: sdk/${{ parameters.ServiceDirectory }}
+          CheckLinkGuidance: $true
```

Comment:
I suspect this might also need to be set for the aggregate-reports pipeline.

Reply:
I did not enable aggregate-report yml. Will trigger the aggregate-report again once I check in the changes.

Reply:
I'm Ok with that as long as you quickly follow-up on any broken links as I want to make sure we don't accidently break any pipelines.

Reply:
I plan to work on lang repos subsequently as work altogether will cause a spike unfortunately.